### PR TITLE
bitwarden: Fix KeyError in search_field (#8549)

### PR DESCRIPTION
**Pull request by sistason**
##### SUMMARY
Fix the KeyError in #8549.

(no changelog fragment needed for this, right?)

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
bitwarden

### CODE REVIEW

**Review by felixfontein — COMMENTED** [review]
Thanks for fixing this! Can you add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/collection_development_process.html#creating-a-changelog-fragment)? Thanks.

File: changelogs/fragments/8557-fix-bug-with-bitwarden.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - "bitwarden - fix KeyError in search_field (https://github.com/ansible-collections/community.general/pull/8557)."
```
Comment (felixfontein) [inline changelogs/fragments/8557-fix-bug-with-bitwarden.yml:2]:
```suggestion
  - "bitwarden lookup plugin - fix ``KeyError`` in ``search_field`` (https://github.com/ansible-collections/community.general/pull/8557)."
```

**Comment by sistason** [comment]
thank you for helping with the changelog fragments.

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@sistason thanks for your contribution!

**Pull request merged**